src/ui/menu_nbgl.c

```diff
@@ -49,7 +50,10 @@ static bool nav_callback(uint8_t page, nbgl_pageContent_t* content) {
 }
 
 void ui_menu_about() {
-    nbgl_useCaseSettings(APPNAME, 0, 1, false, ui_menu_main, nav_callback, NULL);
+#define TOTAL_PAGE_NB        (1)
+#define INIT_PAGE_INDEX      (0)
+#define DISABLE_SUB_SETTINGS (false)
```

Comment:
What do you think of:
```suggestion
#define IS_SUB_SETTINGS (false)
```

Reply:
same comment

Reply:
indeed!